charts/opentelemetry-demo/README.md

```diff
@@ -82,6 +82,7 @@ the demo
 | `default.image.tag`                    | Demo components image tag (leave blank to use app version)                                | `nil`                                                |
 | `default.image.pullPolicy`             | Demo components image pull policy                                                         | `IfNotPresent`                                       |
 | `default.image.pullSecrets`            | Demo components image pull secrets                                                        | `[]`                                                 |
+| `default.replicas`                     | Number of replicas for each component                                                     | `1`                                                  |
```

Comment:
How about adding a note that shared  components such as `Redis, Postgres, and Kafka` should be set by `replicas` under `components`?

Reply:
I set a note at the component level in the docs to mention this